docker-compose.yml

```diff
@@ -75,6 +75,25 @@ services:
       - app_static:/app/static:ro
       - app_socket:/socket
 
+  deployment:
+    container_name: deployment_app
+    profiles: [ prod ]
+    build:
+      context: deployment_service
+      dockerfile: Dockerfile
+      target: runtime
+    image: "ghcr.io/mrlvsb/deployment:latest"
+    pull_policy: always
+    restart: unless-stopped
+    environment:
+      - DOCKER__COMPOSE_FILE_PATH=${DOCKER__COMPOSE_FILE_PATH}
+      - SECURITY__WEBHOOK_SECRET=${SECURITY__WEBHOOK_SECRET}
+      - SECURITY__BACKEND_CORS_ORIGINS=${SECURITY__BACKEND_CORS_ORIGINS}
+      - SECURITY__ALLOWED_HOSTS=${SECURITY__ALLOWED_HOSTS}
+    volumes:
+      - ${REPO__DIRECTORY_PATH}:/kelvin
```

Comment:
Please remind me what is this for? I haven't found any mention of `/kelvin` in the deployment service.